README.md

```diff
@@ -0,0 +1,134 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Postgresql Databases ](#-postgresql-databases-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Postgresql Databases <a name="about-project"></a>
+
+**[Postgresql Databses]** is:
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project is built using.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+ <li><a href="https://rubyonrails.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+> The key features of the project are :
+
+- **[ Decode morsecode using ruby]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project is:
+
+- [un-availbale]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+```
+git clone git@github.com:MohamedCK/postgresql.git
+```
+
+```
+
```

Comment:
- [ ] Kindly remove these three backticks as it is causing your readme file not to be presented well from this line downward.

  ![image](https://user-images.githubusercontent.com/54848363/209337226-3f51df9e-31de-4d41-9b20-eb672a4012fc.png)

Reply:
> * [ ]  Kindly remove these three backticks as it is causing your readme file not to be presented well from this line downward.
>   ![image](https://user-images.githubusercontent.com/54848363/209337226-3f51df9e-31de-4d41-9b20-eb672a4012fc.png)

- [ ] As the previous reviewer suggested, kindly fix the issue.
Currently, it looks:
![readme](https://user-images.githubusercontent.com/70483835/209343694-b4bb409b-d547-4eff-84cc-af71eddede6b.png)